lib/wasix/src/syscalls/wasi/clock_time_get.rs

```diff
@@ -11,8 +14,6 @@ use crate::syscalls::*;
 /// Output:
 /// - `Timestamp *time`
 ///     The value of the clock in nanoseconds
-//#[instrument(level = "trace", skip_all, fields(?clock_id, %precision), ret)]
-#[instrument(level = "trace", skip_all, ret)]
```

Comment:
If it's useful, we can keep this instrumentation behind an additional feature flag, like

```
#[cfg_attr(feature = "extra-logging", tracing::instrument(...)]
```